backend/services/EventHandler.py

```
@@ -30,23 +30,17 @@
 
 async def handle_event_upload(
     media_file: Optional[UploadFile],
     folder_name: str
 ) -> Dict[str, str]:
-    """Handle media file upload to cloudinary and return upload details."""
-    if not media_file:
-        return {"secure_url": None}
-    return await upload_to_cloudinary(media_file, folder_name)
+    upload_result = upload_to_cloudinary(
+        media_file.file,
+        folder_name=folder_name
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)